src/Controls/PlanarControls.js

```diff
@@ -2,7 +2,7 @@ import * as THREE from 'three';
 import { MAIN_LOOP_EVENTS } from 'Core/MainLoop';
 
 // event keycode
-const keys = {
+export const keys = {
```

Comment:
why you export `keys`?

Reply:
Exporting `keys` allows unit tests to access it.